23_Arithmetic2UnitTest/app_resources/shaderCommon.hlsl

```diff
@@ -0,0 +1,68 @@
+#include "common.hlsl"
+
+#include "nbl/builtin/hlsl/glsl_compat/core.hlsl"
+#include "nbl/builtin/hlsl/subgroup/basic.hlsl"
+#include "nbl/builtin/hlsl/subgroup/arithmetic_portability.hlsl"
+#include "nbl/builtin/hlsl/subgroup2/arithmetic_portability.hlsl"
+
+#include "nbl/builtin/hlsl/jit/device_capabilities.hlsl"
+
+// https://github.com/microsoft/DirectXShaderCompiler/issues/6144
+uint32_t3 nbl::hlsl::glsl::gl_WorkGroupSize() {return uint32_t3(WORKGROUP_SIZE,1,1);}
+
+#ifndef ITEMS_PER_INVOCATION
+#error "Define ITEMS_PER_INVOCATION!"
+#endif
+
+typedef vector<uint32_t, ITEMS_PER_INVOCATION> type_t;
+
+// unfortunately DXC chokes on descriptors as static members
+// https://github.com/microsoft/DirectXShaderCompiler/issues/5940
+[[vk::binding(0, 0)]] StructuredBuffer<type_t> inputValue;
+[[vk::binding(1, 0)]] RWByteAddressBuffer output[8];
```

Comment:
can you make the test use BDA so its simpler (no descriptor sets, just BDA sent via push constants)